src/Entity/Organization.php

```diff
@@ -0,0 +1,132 @@
+<?php
+declare(strict_types=1);
+
+namespace AppBundle\Entity;
+
+use Symfony\Component\Validator\Constraints as Assert;
+
+class Organization
+{
+    private $id;
+    private $name;
+    private $address;
+    private $logo;
+
+    private $orderLeadTime;
+    private $limitHourOrder;
+    private $startHourOrder;
+    private $group;
+    private $dayOfOrderAvailable;
+    private $numberOfOrderAvailable;
+    private $amountOfSubsidyPerEmployeeAndOrder;
+    private $coverageOfDeliveryCostsByTheCompanyOrTheEmployee;
+    /**
+     * @var string
+     *
+     * @Assert\Type(type="string")
+     */
+    protected $deliveryPerimeterExpression = 'distance < 3000';
```

Comment:
I would prefer to have all these fields in an `OrganizationConfig` entity/table.

Reply:
ok